src/ui/ToggleEditor.tsx

```diff
@@ -191,12 +191,6 @@ class ToggleEditor extends Component<ToggleEditorProps, ToggleEditorState> {
     super(props);
 
     this.cmOptions = Object.assign(defaultCmOptions, props.cmOptions);
-    this.language = props.language;
-    this.parse = this.language.parse;
-    this.getExceptionMessage = this.language.getExceptionMessage;
-    this.getASTNodeForPrimitive = this.language.getASTNodeForPrimitive;
-    this.getLiteralNodeForPrimitive = this.language.getLiteralNodeForPrimitive;
-    this.primitivesFn = this.language.primitivesFn;
```

Comment:
You should remove the corresponding type declarations for these properties on line 169-174. If you add `"strictPropertyInitialization": true` to the tsconfig.json file, it will complain about these things. If you forget to remove the type declarations, then typescript won't complain when you try to access `this.language` somewhere else in the component.

I didn't turn on any of the stricter typing features because it would have required refactoring a lot more code to get typescript to compile, but turning on stricter typing is a good idea longer term.

Reply:
Done